test/components/remote_retro_test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { shallow } from "enzyme"
+import { expect } from "chai"
+import RemoteRetro from "../../web/static/js/components/remote_retro"
+
+describe("<RemoteRetro>", () => {
+  const mockRetroChannel = {
+    on: () => {},
+    join: () => {},
+  }
+  context("when the current user is a facilitator", () => {
+    const presences = {
+      userToken: {
+        user: { facilitator: true },
+      },
+    }
+
+    it("renders a room with isFacilitiator true", () => {
+      const wrapper = shallow(
+        <RemoteRetro userToken="userToken" retroChannel={mockRetroChannel} />)
+
+      wrapper.setState({ presences })
```

Comment:
Travis and I have been talking about moving away from setting state in tests. We'd probably want to introduce something like redux to handle the channel callbacks. Currently different callbacks are being declared to set state on several difference components. Ideally we'd change applications state and pass the props only to components that need to react to the change.